src/main/scala/DataMunging.scala

```diff
@@ -0,0 +1,40 @@
+/**
+  * Created by idarlington on 2/26/17.
+  */
+object DataMunging {
+
+  case class weatherLine(
```

Comment:
Thanks, I decided earlier to I need to put in more effort to using a style in coding. Thanks for the correction.